src/confluent_kafka/src/Producer.c

```diff
@@ -879,7 +896,7 @@ PyTypeObject ProducerType = {
 	0,                         /*tp_setattr*/
 	0,                         /*tp_compare*/
 	0,                         /*tp_repr*/
-	0,                         /*tp_as_number*/
+	&Producer_num_methods,                         /*tp_as_number*/
```

Comment:
Try to keep comment in line with other comments.

Reply:
I aligned the nearby `seq_methods` as well, let me know if you want it reverted.